bundle/src/test/java/com/adobe/acs/commons/it/build/ScrMetadataIT.java

```diff
@@ -275,95 +229,137 @@ private DescriptorList getDescriptorsFromCurrent() throws Exception {
     private DescriptorList parseJar(InputStream is, boolean checkNs) throws Exception {
         DescriptorList result = new DescriptorList();
 
+        List<Descriptor> metatypeDescriptors = new LinkedList<>();
         try (ZipInputStream zis = new ZipInputStream(is)) {
             ZipEntry entry = zis.getNextEntry();
             while (entry != null) {
                 if (!entry.isDirectory() && entry.getName().endsWith(".xml")) {
                     if (entry.getName().startsWith("OSGI-INF/metatype")) {
-                        result.merge(parseMetatype(new InputStreamFacade(zis), entry.getName()));
+                        metatypeDescriptors.add(parseMetatype(new InputStreamFacade(zis), entry.getName()));
                     } else if (entry.getName().startsWith("OSGI-INF/")) {
                         result.merge(parseScr(new InputStreamFacade(zis), entry.getName(), checkNs));
                     }
                 }
-
                 entry = zis.getNextEntry();
             }
         }
+        // metatype descriptors must come last (after component descriptions)
```

Comment:
@justinedelson Any reason why you think that metatype attributes should overwrite the value of same named component properties? IMHO those two are not directly related. Currently it feels a bit like we a comparing apples and pears. The correct way would be to compare SCR descriptors and metatype descriptors separately, as on affects the default properties of DS components, the other one only how the Felix Web Console renders a UI for setting custom properties.